Actions/PullRequestClosedAction.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+using Slack.Json.Github;
+using Slack.Json.Slack;
+using Slack.Json.Util;
+
+namespace Slack.Json.Actions
+{
+    public class PullRequestClosedAction : IRequestAction
+    {
+        public string GithubHookEventName => "pull_request";
+        public string SlackJsonType => "pull_request";
+        private readonly ISlackMessaging slack;
+        private readonly ILogger<PullRequestClosedAction> logger;
+
+        public PullRequestClosedAction(ISlackMessaging slack, ILogger<PullRequestClosedAction> logger)
+        {
+            this.slack = slack;
+            this.logger = logger;
+        }
+
+        public void Execute(JObject request, IEnumerable<ISlackAction> actions)
+        {
+            if(request.Get<string>(x => x.action) != "closed")
+                return;
+
+            ActionUtils.ParsePullRequestDefaultFields(request, out var prHtmlUrl, out var prTitle);
+
+            var isMerged = !string.IsNullOrEmpty(request.Get(x => x.pull_request.merged_at));
+
+            var color = isMerged ?  "#6F42C1" : "danger";
```

Comment:
I would prefer using constants with these colors to have unified look and to avoid having to learn what color is #6F42C1.